kumascript/src/api/util.ts

```diff
@@ -4,15 +4,11 @@
  */
 import sanitizeFilename from "sanitize-filename";
 import * as cheerio from "cheerio";
+import chalk from "chalk";
 
 const H1_TO_H6_TAGS = new Set(["h1", "h2", "h3", "h4", "h5", "h6"]);
 const HEADING_TAGS = new Set([...H1_TO_H6_TAGS, "hgroup"]);
-const INJECT_SECTION_ID_TAGS = new Set([
-  ...HEADING_TAGS,
-  "section",
-  "div",
-  "dt",
-]);
+const INJECT_SECTION_ID_TAGS = new Set([...HEADING_TAGS, "section", "dt"]);
```

Comment:
FYI Here's the commit that added `"div"` here: https://github.com/mdn/yari/pull/5190/commits/720711732280d240b2f082dd648bd5a2fd9735d7

Reply:
Thanks. I don't think there's a practical use. Those test show some good legacy and we'll remove them soon.